content/07.pathogenesis.md

```diff
@@ -79,16 +87,14 @@ In Iran, estimates of R~t~ declined from 4.86 in the first week to 2.1 by the fo
 In Europe, analysis of 11 countries inferred the dynamics of R~t~ over a time range from the beginning of the outbreak until March 28, 2020, by which point most countries had implemented major interventions (such as school closures, public gathering bans, and stay-at-home orders) [@doi:10.25561/77731].
 Across all countries, the mean R~t~ before interventions began was estimated as 3.87; R~t~ varied considerably, from below 3 in Norway to above 4.5 in Spain.
 After interventions, R~t~ decreased by an average of 64% across all countries, with mean R~t~=1.43.
-The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden (note that this is in part because Sweden did not implement social distancing measures on the same scale as other countries).
+The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden, this may be in part because Sweden did not implement social distancing measures on the same scale as other countries.
```

Comment:
👍

Reply:
tiny grammar typo so not a big deal if you don't get to it:
if you want to replace parentheses with comma you need to put a "--" or a "which" so that it's not 2 independent clauses.
"The lowest predicted value was 0.97 for Norway and... 2.64... for Sweden, which may be in part because..."
or
"The lowest predicted value was 0.97 for Norway and... 2.64... for Sweden -- this may be in part because..."